creativecommons/search
# Removed non-functional menu button

## Fixes



- Fixes #237  by @SisiVero 

## Description
On mobile screens, there is a menu button that is not functional, clicking on that button does nothing to the website and has no content. This pull request removes that button.

## Tests
1. Open the CC's search website
2. Resize the browser to smaller screens or view on a mobile phone
3. Menu button should no longer be there.

## Screenshots
Problem(before)
<img width="317" alt="Screenshot 2024-10-07 at 17 18 59" src="https://github.com/user-attachments/assets/88ecfd37-93b2-4641-9392-f3e2dc67466d">

Solution(after)
<img width="327" alt="Screenshot 2024-10-07 at 17 16 23" src="https://github.com/user-attachments/assets/267136c7-27d9-4d45-8879-0a7f117f16e8">


## Checklist



  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin




For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
